.github/workflows/ci.yml

```diff
@@ -1,14 +1,8 @@
 name: CI
 
 env:
-  JAVA8: adopt@1.8
-  JAVA11: adopt@1.11
-  SCALA_211: 2.11.12
-  SCALA_212: 2.12.13
-  SCALA_213: 2.13.5
-  SCALA_3: 3.0.0-RC1
-  JAVA_OPTS: -Xms2048M -Xmx3072M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
-  JVM_OPTS:  -Xms2048M -Xmx3072M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
```

Comment:
@mijicd just checking... all those options need to be there, `-Xmx` wouldn't be enough, right?

Reply:
They can probably be pruned, as they were ported from CircleCI setup. However, note that they were introduced there using the good old trial and error :).

Reply:
So I've kept only `-Xmx` and it seems to be working OK :crossed_fingers:

Reply:
Thanks for optimizing this as long as we maintain pipeline stability I am happy. Completely agree with keeping the `-Xmx` option as that made the difference when I was tweaking this last.